TeaDiary.business/Interfaces/IUserProvider.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+
+namespace TeaDiary.business.Interfaces
+{
+    public interface IUserProvider
```

Comment:
I have no idea what specific user you're talking about

Reply:
Changed name for both interface as well as implementation. Will look through the code before commiting to get sure everything is ok